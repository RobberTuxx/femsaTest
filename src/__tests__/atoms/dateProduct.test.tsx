import React from 'react';
import renderer from 'react-test-renderer';
import DateProduct from "../../components/atoms/dateProduct";
it('render date product', ()=> {
    const component = renderer.create(<DateProduct text={"2023-03-11T12:17:47.118Z"}/>).toJSON()
    expect(component).toMatchSnapshot()
})
it('has 0 child', ()=> {
    const component = renderer.create(<DateProduct text={"2023-03-11T12:17:47.118Z"}/>).toJSON()
    expect(component.children.length).toBe(1)
})
