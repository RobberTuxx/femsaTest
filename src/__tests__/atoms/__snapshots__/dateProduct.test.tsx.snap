// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render date product 1`] = `
<Text
  style={
    {
      "color": "#000000",
      "fontFamily": "Avenir",
      "fontSize": 19,
      "fontWeight": "400",
    }
  }
>
  11 de marzo de 2023
</Text>
`;
